t/39-scheduled_products-table.t

```diff
@@ -9,8 +9,10 @@ use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
 use Test::Mojo;
 use Test::Warnings ':report_warnings';
 use Test::MockModule;
+use Test::MockObject;
 use OpenQA::Jobs::Constants;
 use OpenQA::Schema::Result::ScheduledProducts qw(ADDED SCHEDULED SCHEDULING CANCELLING CANCELLED);
+use OpenQA::Task::SignalGuard;
```

Comment:
Please use some kind of fake object in *all* these tests. We probably don't want to influence the signal handling of unit tests.